model/src/trade.rs

```diff
@@ -0,0 +1,23 @@
+//! Contains the Trade type as described by the specification with serialization as described by the openapi documentation.
```

Comment:
Imo it would be good to have a real json serialization test like `deserialization_and_back()`  in `model/src/order.rs` so that we can be sure it matches what we say in openapi.

Reply:
Done! and nice catch, I had forgotten the switch to `camelCase`